nixos/lib/eval-config.nix

```diff
@@ -22,8 +22,17 @@
   check ? true
 , prefix ? []
 , lib ? import ../../lib
-}:
+}@args: if !(builtins ? modulePath) then let
```

Comment:
Not on the initial import, but on the scoped reimport there will be! This avoids an infinte loop of reimports.